integration/apps/sinatra2_classic/spec/integration/basic_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require 'rspec/wait'
+require 'securerandom'
+require 'json'
+
+RSpec.describe 'Basic scenarios' do
+  include_context 'integration test'
+
+  context 'default' do
+    subject { get('basic/default') }
+    it { is_expected.to be_a_kind_of(Net::HTTPOK) }
+  end
+end
```

Comment:
Can I ask you to include the profiler tests here as well? (If it's giving you trouble, let me know, but afaik it should only be copy-pasting the specs from the other example apps)